sdk/src/trace/always_on_sampler.cc

```diff
@@ -0,0 +1,28 @@
+#include "opentelemetry/sdk/trace/always_on_sampler.h"
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace trace
+{
+/**
+ * Return RECORD_AND_SAMPLE and an empty set of attributes
+ */
+SamplingResult AlwaysOnSampler::ShouldSample(
+    const SpanContext *parent_context,
+    trace_api::TraceId trace_id,
+    nostd::string_view name,
+    trace_api::SpanKind span_kind,
+    const trace_api::KeyValueIterable &attributes) noexcept
+{
+  return{sdk::trace::Decision::RECORD_AND_SAMPLE, nullptr};
```

Comment:
Like with the always_off PR, should we just inline the two lines of implementation into the header file?

Reply:
Agreed. I will change it to inline functions.